runhouse/resources/hardware/cluster.py

```diff
@@ -489,8 +517,9 @@ def connect_server_client(self, force_reconnect=False):
             )
             self.client_port = self._rpc_tunnel.local_bind_port
 
-            ssh_user = self.ssh_creds.get("ssh_user")
-            password = self.ssh_creds.get("password")
+            ssh_creds = self.ssh_creds.values
```

Comment:
prob will not need the .values, if self.ssh_creds will return self._ssh_creds.values